pages/guias/PSF.rst

```diff
@@ -0,0 +1,41 @@
+Membresia de la Python Software Foundation
+==========================================
+
+Abrimos el navegador de preferencia y nos dirigimos a:
+
+ `Enlace <https://www.python.org/accounts/signup/>`_
+
+En la ventanana del navegador nos fijamos en el boton Sign Up le damos clic y vemos  que tenemos las siguientes opciones
+
+  * Username : El nombre del usuario con el que deseamos registrarnos
+
+  * Email : El nombre del usuario con el que deseamos registrarnos
+
+  * Password : La clave con la que vamos a acceder a la pagina de la P.S.F.
+
+  **En la pestana nos indica un texto: Confirmation email send to xxxxx@xxxxx.com con el email que nos registramos**
+
+En el navegador web abrimos una una nueva pestana e ingresamos al email con que nos registramos
+
+Abierto el email nos dirigimos a la bandeja de entrada y verificamos el email de la P.S.F. ,caso contrario ir a la carpeta Spam
+
+En el mensaje nos indica::
```

Comment:
Hola, veo que no ha pasado las pruebas de CI, quiero sugerir algo, estoy aprendiendo a usar github espero pueda hacerme entender. En está parte del texto _línea 22 a la 31_ que muestra el mensaje de la PSF que llega al correo; está bien mostrarlo en una nota o tipo caja `topic`, pero me parece que la sintaxis seria así.
```rst
.. topic:: En el mensaje nos indica:

    Hello from python.org!

    You're receiving this e-mail because user xxxxxx at python.org has given yours as an e-mail address to connect their account.

    To confirm this is correct, go to  `<http://www.python.org/accounts/confirm-email/xxxxxxxxxxxxxxxxxxxxxxxx>`_

    Thank you from python.org!
    python.org
```

O también se podría evitar dar formato de vinculo a la URL en la línea 28 para evitar este error; además al ser link que simboliza o representa el original, dentro del contexto del tutorial puede no ser tan necesario.

Espero sea de ayuda.

Reply:
Y disculpas, no supe como señalar el comentario en todas esas líneas.

Reply:
Gracias ,por tu ayuda

Reply:
@rpalaciosg muy bien el review!

> no supe como señalar el comentario en todas esas líneas

Es una de las restricciones de GitHub, sólo se puede hacer un comentario sobre una linea.

Reply:
De nada @gomezgleonardob me alegra haber ayudado.
y gracias @stsewd estar trabajando en el mini tutorial de `reestructuredtext` me ayudó. 
No sé si es por la poca experiencia en _rst_, pero creo que aun prefiero `markdown`.

Reply:
reStructured text tiene una syntaxis rara, pero es mucho más estándar y extensible que markdown

Reply:
Debe ser tu que tienes mas experiencia, con todo voy a seguirlo probando y usando para ganar mas experiencia.